src/user/scheduling.md

```diff
@@ -319,3 +319,99 @@ Some aspects of the meta data synchronization feature to be aware of:
 
 - You can see the time of last successful synchronization with remote
   server in the scheduling screen next to the "Last success" label.
+
+### Predictor { #scheduling_predictor } 
+
+This runs selected predictors and/or predictor groups.
+
+The relative start and relative end parameters determine the periods in
+which data will be predicted, relative to the date on which the
+predictor job is run:
+
+- **Relative start** counts the days from the job date to the
+earliest date on which a predicted period may start. It can be positive
+or negative. For example, a value of 3 means predict into periods that
+start at least 3 days after the predictor run. A value of -3 means
+predict into periods that start at least 3 days before the predictor run.
+
+- **Relative end** counts the days from the job date to the
+latest date on which a predicted period may end. It can be positive
+or negative. For example, a value of 9 means predict into periods that
+end at least 9 days after the predictor run. A value of -9 means
+predict into periods that end at least 9 days before the predictor run.
+
+Setting these values can give you very flexible control over when
+predictions will be made, especially if your predictor job is set to
+run daily or more frequently. Before you set these values, you should
+think carefully about when you want predictions for a period to start
+being made, and when you want them to stop being made. Then you
+need to compute the appropriate relative start
+and end dates.
+
+Examples:
+
+1. A predictor is computed from weekly data of the same week. (No past
+sampled data is used.) After the week ends on Sunday,
+you expect the weekly data to be entered on the Monday and Tuesday.
```

Comment:
Maybe "the following Monday and Tuesday" is clearer?

Reply:
Changed to "After the week ends on Sunday, you expect the data to be entered in the following two days: Monday and Tuesday."

Reply:
This works fine, but I would suggest, "the following two days (Monday and Tuesday)."

Reply:
Perfect, thanks!